src/app/test-record/test-section/edit/test-section-edit.component.scss

```diff
@@ -0,0 +1,13 @@
+.govuk-link,
+.govuk-link:visited {
+  color: #1d70b8 !important;
+  font-weight: 500;
+  cursor: pointer;
+  text-decoration: underline;
+  line-height: 2rem;
+  float: right;
+}
+
+.govuk-checkboxes__label {
```

Comment:
If we have to tweak the text because the content is not wrapping, let's create our own util class and not override the existing lib